lang/python/python3-find-stdlib-depends.sh

```diff
@@ -0,0 +1,434 @@
+#!/bin/sh
+
+# Packages data
+#
+# Notes:
+# * python3-codecs: Also contains codecs for CJK encodings but we don't
+#   have a good way to check for uses
+# * python3-openssl: Don't include hashlib as it supports several
+#   standard algorithms without requiring OpenSSL
+
+packages="
+python3-asyncio: asyncio
+python3-cgi: cgi
+python3-cgitb: cgitb
+python3-codecs: unicodedata
+python3-ctypes: ctypes
+python3-dbm: dbm
+python3-decimal: decimal
+python3-distutils: distutils
+python3-email: email
+python3-gdbm: dbm.gnu
+python3-logging: logging
+python3-lzma: lzma
+python3-multiprocessing: multiprocessing
+python3-ncurses: ncurses
+python3-openssl: ssl
+python3-pydoc: doctest pydoc
+python3-sqlite3: sqlite3
+python3-unittest: unittest
+python3-urllib: urllib
+python3-xml: xml xmlrpc
```

Comment:
this could also add [at some point] a logic to detect if new python standard packages are added/removed;

Reply:
I wish there was a way to synchronize/share information between the makefiles and this script; I couldn't think of an easy way to do this.

Reply:
well, it should be possible; but no need to do it now;

if you do `ls lang/python/python3/files/python3-package-*.mk` you get more-or-less the list of packages in this list;

then, there are various ways to go from here;
probably the best is to grep for `Py3BasePackage`; and get out the name/key [e.g. python3-xml for example]

the table-matching [i.e. `python3-xml: xml xmlrpc`] is fine to leave here as-is;
we only need to get a message if we need to update the table; no-more-no-less;
if there are some packages we need to ignore we can do `python3-pkg: <ignore>` or some other keyword;

but as i said; we can definitely leave it for later; things don't change too often;
this is great as-is